### PR TITLE
[FIX] XMLTemplateAnalyzer: Handle empty XML view/fragment

### PR DESCRIPTION
**Pull request by matz3**
Fixes: https://github.com/SAP/ui5-builder/issues/281

**Thank you for your contribution!** 🙌

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ ] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#-contributing-code)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-tooling/blob/master/CONTRIBUTING.md#how-to-contribute) section 
- [ ] [No merge commits](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#no-merge-commits)
- [ ] [Correct commit message style](https://github.com/SAP/ui5-tooling/blob/master/docs/Guidelines.md#commit-message-style)

### CODE REVIEW

File: lib/lbt/analyzer/XMLTemplateAnalyzer.js [hunk 1]
```diff
@@ -162,6 +162,13 @@ class XMLTemplateAnalyzer {
 					return;
 				}
 
+				if ( !result ) {
+					// Skip empty xml views/fragments
+					log.warn("Ignoring empty file " + info.name);
```
[the remark this answers is not in the document]
Comment (matz3) [reply lib/lbt/analyzer/XMLTemplateAnalyzer.js:167]: Makes sense. I will change this to throw a proper error

[the remark this answers is not in the document]
Comment (matz3) [reply lib/lbt/analyzer/XMLTemplateAnalyzer.js]: Done

File: test/lib/lbt/analyzer/XMLTemplateAnalyzer.js [hunk 2]
```diff
@@ -193,13 +193,13 @@ test("integration: Analysis of an empty xml view", async (t) => {
 		};
```
Comment (codeworrior) [inline test/lib/lbt/analyzer/XMLTemplateAnalyzer.js:193]: This looks suspicious to me. I would a) expect that JSON.stringify does the wrong thing for XML content (e.g. add surrounding quotes, escape quotes in the content) and b) that due to a) the error message should be different, not the "Invalid empty XML" message.

What am I missing?

Comment (matz3) [reply test/lib/lbt/analyzer/XMLTemplateAnalyzer.js]: Done.

I also wondered about that but just copied the existing tests without thinking about it. The mockPool doesn't seem to be used to read the XML content. It just needs a findResource function that resolves.

**Pull request merged**